src/store/RelayPendingQueryTracker.js

```diff
@@ -118,12 +118,14 @@ class PendingFetch {
   _errors: Array<?Error>;
 
   constructor(
+    /* eslint-disable comma-dangle */
```

Comment:
So really the issue is that the `ObjectPattern` loc is also accounting for the typeAnnotation so the column start and column end are 2 and 3 so it thinks it is actually multiline. Not sure what we can do other than making a forked rule in https://github.com/babel/eslint-plugin-babel. Or we would have to change the range/loc of nodes in either babel/babel-eslint.